home.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:io';
+import 'Login.dart';
+import 'Sign-up.dart';
+
+void main() {
+  home main = new home();
+}
+
+class home {
+  home() {

```

Comment:
in the init (constructor) function we usually initiate resources or do some preparations, we shouldn't have any business logic.

Please move to another function st. `start()` or `run()` and call it after creating the home object in the main method.